tqdm/keras.py

```diff
@@ -69,10 +69,12 @@ def __init__(self, epochs=None, data_size=None, batch_size=None, verbose=1,
     def on_train_begin(self, *_, **__):
         params = self.params.get
         auto_total = params('epochs', params('nb_epoch', None))
-        if auto_total is not None:
+        if auto_total is not None and auto_total != self.epoch_bar.total:
             self.epoch_bar.reset(total=auto_total)
 
-    def on_epoch_begin(self, *_, **__):
+    def on_epoch_begin(self, epoch, *_, **__):
+        if self.epoch_bar.n < epoch:
+            self.epoch_bar.update(epoch-self.epoch_bar.n)
```

Comment:
Is this only ever triggered exactly once when training is resumed? Or are there any other circumstances?

Reply:
Yes, exactly once. I don't figure out other circumstances.